<!DOCTYPE html>
<html>

  <!-- Begin Jekyll SEO tag v2.8.0 -->
<title>microbial genomics | The Jiang Lab at NLM</title>
<meta name="generator" content="Jekyll v3.9.2" />
<meta property="og:title" content="microbial genomics" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Yang Y, Jiang X. 2023. Evolink: A Phylogenetic Approach for Rapid Identification of Genotype-Phenotype Associations in Large-scale Microbial Multi-Species Data. Bioinformatics. DOI: 10.1093/bioinformatics/btad215. Hall B, Levy S, Dufault-Thompson K, Ndjite GM, Weiss A, Braccia D, Jenkins C, Yang Y, Arp G, Abeysinghe S, Jermain M, Wu CH, Jiang X. 2023. Discovery of the gut microbial enzyme responsible for bilirubin reduction to urobilinogen. bioRxiv. DOI: 10.1101/2023.02.07.527579. Braccia D J, Ndjite G M, Weiss A, Levy S, Abeysinghe S, Jiang X, Pop M, Hall B. 2022. Gut microbiome-wide search for bacterial azoreductases reveals potentially uncharacterized azoreductases encoded in the human gut microbiome. Drug Metabolism and Disposition. DOI: 10.1124/dmd.122.000898. Dufault-Thompson K, Hall AB, Jiang X. 2022. Taxonomic Distribution and Evolutionary Analysis of the Equol Biosynthesis Gene Cluster. BMC Genomics. 23: 1. DOI: 10.1186/s12864-022-08426-7. Zhang X, Wang R, Xie X, Hu Y, Wang J, Sun Q, Feng X, Lin W, Tong S, Yan W, Wen H, Wang M, Zhai S, Sun C, Wang F, Niu Q, Kropinski A M, Cui Y, Jiang X, Peng S, Li S, Tong Y. 2022. Mining bacterial NGS data vastly expands the complete genomes of temperate phages. NAR genomics and bioinformatics. 4: lqac057. DOI: 10.1093/nargab/lqac057. Yan W, Hall AB, Jiang X. 2022. Bacteroidales Species in the Human Gut Are a Reservoir of Antibiotic Resistance Genes Regulated by Invertible Promoters. Npj Biofilms and Microbiomes. 8: 1. DOI: 10.1038/s41522-021-00260-1. Olejniczak M, Jiang X, Basczok MM, Storz G. 2022. KH Domain Proteins: Another Family of Bacterial RNA Matchmakers. Molecular Microbiology. 117: 10-19. DOI: 10.1111/mmi.14842. Mou Z, Yang Y, Hall AB, Jiang X. 2021. The Taxonomic Distribution of Histamine-Secreting Bacteria in the Human Gut Microbiome. BMC Genomics. 22: 695. DOI: 10.1186/s12864-021-08004-3. Braccia DJ, Jiang X, Pop M, Hall AB. 2021. The Capacity to Produce Hydrogen Sulfide (H2S) via Cysteine Degradation Is Ubiquitous in the Human Gut Microbiome. Frontiers in Microbiology. 12: 705583. DOI: 10.3389/fmicb.2021.705583." />
<meta property="og:description" content="Yang Y, Jiang X. 2023. Evolink: A Phylogenetic Approach for Rapid Identification of Genotype-Phenotype Associations in Large-scale Microbial Multi-Species Data. Bioinformatics. DOI: 10.1093/bioinformatics/btad215. Hall B, Levy S, Dufault-Thompson K, Ndjite GM, Weiss A, Braccia D, Jenkins C, Yang Y, Arp G, Abeysinghe S, Jermain M, Wu CH, Jiang X. 2023. Discovery of the gut microbial enzyme responsible for bilirubin reduction to urobilinogen. bioRxiv. DOI: 10.1101/2023.02.07.527579. Braccia D J, Ndjite G M, Weiss A, Levy S, Abeysinghe S, Jiang X, Pop M, Hall B. 2022. Gut microbiome-wide search for bacterial azoreductases reveals potentially uncharacterized azoreductases encoded in the human gut microbiome. Drug Metabolism and Disposition. DOI: 10.1124/dmd.122.000898. Dufault-Thompson K, Hall AB, Jiang X. 2022. Taxonomic Distribution and Evolutionary Analysis of the Equol Biosynthesis Gene Cluster. BMC Genomics. 23: 1. DOI: 10.1186/s12864-022-08426-7. Zhang X, Wang R, Xie X, Hu Y, Wang J, Sun Q, Feng X, Lin W, Tong S, Yan W, Wen H, Wang M, Zhai S, Sun C, Wang F, Niu Q, Kropinski A M, Cui Y, Jiang X, Peng S, Li S, Tong Y. 2022. Mining bacterial NGS data vastly expands the complete genomes of temperate phages. NAR genomics and bioinformatics. 4: lqac057. DOI: 10.1093/nargab/lqac057. Yan W, Hall AB, Jiang X. 2022. Bacteroidales Species in the Human Gut Are a Reservoir of Antibiotic Resistance Genes Regulated by Invertible Promoters. Npj Biofilms and Microbiomes. 8: 1. DOI: 10.1038/s41522-021-00260-1. Olejniczak M, Jiang X, Basczok MM, Storz G. 2022. KH Domain Proteins: Another Family of Bacterial RNA Matchmakers. Molecular Microbiology. 117: 10-19. DOI: 10.1111/mmi.14842. Mou Z, Yang Y, Hall AB, Jiang X. 2021. The Taxonomic Distribution of Histamine-Secreting Bacteria in the Human Gut Microbiome. BMC Genomics. 22: 695. DOI: 10.1186/s12864-021-08004-3. Braccia DJ, Jiang X, Pop M, Hall AB. 2021. The Capacity to Produce Hydrogen Sulfide (H2S) via Cysteine Degradation Is Ubiquitous in the Human Gut Microbiome. Frontiers in Microbiology. 12: 705583. DOI: 10.3389/fmicb.2021.705583." />
<link rel="canonical" href="https:///research/bacteria.html" />
<meta property="og:url" content="https:///research/bacteria.html" />
<meta property="og:site_name" content="The Jiang Lab at NLM" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2023-08-10T11:48:33-05:00" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="microbial genomics" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2023-08-10T11:48:33-05:00","datePublished":"2023-08-10T11:48:33-05:00","description":"Yang Y, Jiang X. 2023. Evolink: A Phylogenetic Approach for Rapid Identification of Genotype-Phenotype Associations in Large-scale Microbial Multi-Species Data. Bioinformatics. DOI: 10.1093/bioinformatics/btad215. Hall B, Levy S, Dufault-Thompson K, Ndjite GM, Weiss A, Braccia D, Jenkins C, Yang Y, Arp G, Abeysinghe S, Jermain M, Wu CH, Jiang X. 2023. Discovery of the gut microbial enzyme responsible for bilirubin reduction to urobilinogen. bioRxiv. DOI: 10.1101/2023.02.07.527579. Braccia D J, Ndjite G M, Weiss A, Levy S, Abeysinghe S, Jiang X, Pop M, Hall B. 2022. Gut microbiome-wide search for bacterial azoreductases reveals potentially uncharacterized azoreductases encoded in the human gut microbiome. Drug Metabolism and Disposition. DOI: 10.1124/dmd.122.000898. Dufault-Thompson K, Hall AB, Jiang X. 2022. Taxonomic Distribution and Evolutionary Analysis of the Equol Biosynthesis Gene Cluster. BMC Genomics. 23: 1. DOI: 10.1186/s12864-022-08426-7. Zhang X, Wang R, Xie X, Hu Y, Wang J, Sun Q, Feng X, Lin W, Tong S, Yan W, Wen H, Wang M, Zhai S, Sun C, Wang F, Niu Q, Kropinski A M, Cui Y, Jiang X, Peng S, Li S, Tong Y. 2022. Mining bacterial NGS data vastly expands the complete genomes of temperate phages. NAR genomics and bioinformatics. 4: lqac057. DOI: 10.1093/nargab/lqac057. Yan W, Hall AB, Jiang X. 2022. Bacteroidales Species in the Human Gut Are a Reservoir of Antibiotic Resistance Genes Regulated by Invertible Promoters. Npj Biofilms and Microbiomes. 8: 1. DOI: 10.1038/s41522-021-00260-1. Olejniczak M, Jiang X, Basczok MM, Storz G. 2022. KH Domain Proteins: Another Family of Bacterial RNA Matchmakers. Molecular Microbiology. 117: 10-19. DOI: 10.1111/mmi.14842. Mou Z, Yang Y, Hall AB, Jiang X. 2021. The Taxonomic Distribution of Histamine-Secreting Bacteria in the Human Gut Microbiome. BMC Genomics. 22: 695. DOI: 10.1186/s12864-021-08004-3. Braccia DJ, Jiang X, Pop M, Hall AB. 2021. The Capacity to Produce Hydrogen Sulfide (H2S) via Cysteine Degradation Is Ubiquitous in the Human Gut Microbiome. Frontiers in Microbiology. 12: 705583. DOI: 10.3389/fmicb.2021.705583.","headline":"microbial genomics","mainEntityOfPage":{"@type":"WebPage","@id":"https:///research/bacteria.html"},"url":"https:///research/bacteria.html"}</script>
<!-- End Jekyll SEO tag -->

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width initial-scale=1" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>microbial genomics</title>
    <meta name="description" content="The website for the bioinformatic research group leading by Xiaofang Jiang at the National Library of Medicine, National Institutes of Health.">

	<link rel="stylesheet" type='text/css' href="/css/bootstrap.min.css">
	<link rel="stylesheet" type='text/css' href="/css/font-awesome.min.css">

	<link rel="stylesheet" href="/css/academicons.css" />

	<link rel="stylesheet" type='text/css' href="/css/main.css">

	<link href='https://fonts.googleapis.com/css?family=Ubuntu' rel='stylesheet' type='text/css'>

	<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" rel="stylesheet"  type='text/css'>

    <link rel="canonical" href="https:///research/bacteria.html">
    <link rel="alternate" type="application/rss+xml" title="The Jiang Lab at NLM" href="https:///feed.xml" />

   
</head>


  <body>

	

<div class="header">
	<div class="container-fluid">
		<div class="pull-right">
			<ul class="nav nav-pills">
				
				<li role="presentation" ><a href="/">Home</a></li>
				
				<li role="presentation" class="active"><a href="/research">Research</a></li>
				<li role="presentation" ><a href="/people">Team</a></li>
				<li role="presentation" ><a href="/papers">Papers</a></li>
				<li role="presentation" ><a href="/code">Code</a></li>
				<li role="presentation" ><a href="/contact">Contact</a></li>
			</ul>
		</div>
	</div>
</div>


	<div class="default-wrapper">
	    <div class="index_page">
	<h2>microbial genomics</h2>
    
<div class="papers">
	<ol>
		
			
				<li><div class="reference">

	<b>Yang Y, Jiang X.</b>
	2023.


	
	Evolink: A Phylogenetic Approach for Rapid Identification of Genotype-Phenotype Associations in Large-scale Microbial Multi-Species Data.
	<i>Bioinformatics</i>.
	

	

	

	
	DOI: <a href="http://doi.org/10.1093/bioinformatics/btad215">10.1093/bioinformatics/btad215</a>.
	


	<div class="supp">
		
		<a href="/assets/pdf/2023_Yang.pdf">
			<i class="fa fa-file-pdf-o fa-lg"></i>
		</a>
		
		
		
			
			<a href="https://github.com/nlm-irp-jianglab/Evolink">
				<i class="fa fa-github fa-lg"></i>
			</a>
			
		

		

		

	</div>
</div>
</li>
			
		
			
				<li><div class="reference">

	<b>Hall B, Levy S, Dufault-Thompson K, Ndjite GM, Weiss A, Braccia D, Jenkins C, Yang Y, Arp G, Abeysinghe S, Jermain M, Wu CH, Jiang X.</b>
	2023.


	
	Discovery of the gut microbial enzyme responsible for bilirubin reduction to urobilinogen.
	<i>bioRxiv</i>.
	

	

	

	
	DOI: <a href="http://doi.org/10.1101/2023.02.07.527579">10.1101/2023.02.07.527579</a>.
	


	<div class="supp">
		
		<a href="/assets/pdf/2023_Hall.pdf">
			<i class="fa fa-file-pdf-o fa-lg"></i>
		</a>
		
		
		<a href="https://www.biorxiv.org/content/10.1101/2023.02.07.527579v1">
			<i class="ai ai-biorxiv ai-lg"></i>
		</a>
		
		

		

		

	</div>
</div>
</li>
			
		
			
				<li><div class="reference">

	<b>Braccia D J, Ndjite G M, Weiss A, Levy S, Abeysinghe S, Jiang X, Pop M, Hall B.</b>
	2022.


	
	Gut microbiome-wide search for bacterial azoreductases reveals potentially uncharacterized azoreductases encoded in the human gut microbiome.
	<i>Drug Metabolism and Disposition</i>.
	

	

	

	
	DOI: <a href="http://doi.org/10.1124/dmd.122.000898">10.1124/dmd.122.000898</a>.
	


	<div class="supp">
		
		<a href="/assets/pdf/2022_Braccia.pdf">
			<i class="fa fa-file-pdf-o fa-lg"></i>
		</a>
		
		
		

		

		

	</div>
</div>
</li>
			
		
			
				<li><div class="reference">

	<b>Dufault-Thompson K, Hall AB, Jiang X.</b>
	2022.


	
	Taxonomic Distribution and Evolutionary Analysis of the Equol Biosynthesis Gene Cluster.
	<i>BMC Genomics</i>.
	

	
	<b>23:</b>
	

	
	1.
	

	
	DOI: <a href="http://doi.org/10.1186/s12864-022-08426-7">10.1186/s12864-022-08426-7</a>.
	


	<div class="supp">
		
		<a href="/assets/pdf/2022_Dufault-Thompson_a.pdf">
			<i class="fa fa-file-pdf-o fa-lg"></i>
		</a>
		
		
		

		

		

	</div>
</div>
</li>
			
		
			
				<li><div class="reference">

	<b>Zhang X, Wang R, Xie X, Hu Y, Wang J, Sun Q, Feng X, Lin W, Tong S, Yan W, Wen H, Wang M, Zhai S, Sun C, Wang F, Niu Q, Kropinski A M, Cui Y, Jiang X, Peng S, Li S, Tong Y.</b>
	2022.


	
	Mining bacterial NGS data vastly expands the complete genomes of temperate phages.
	<i>NAR genomics and bioinformatics</i>.
	

	
	<b>4:</b>
	

	
	lqac057.
	

	
	DOI: <a href="http://doi.org/10.1093/nargab/lqac057">10.1093/nargab/lqac057</a>.
	


	<div class="supp">
		
		<a href="/assets/pdf/2022_Zhang.pdf">
			<i class="fa fa-file-pdf-o fa-lg"></i>
		</a>
		
		
		

		
			<a href="https://ftp.ncbi.nlm.nih.gov/pub/mgx/phage/data.zip">
				<i class="fa fa-table fa-lg"></i>
			</a>
		

		

	</div>
</div>
</li>
			
		
			
				<li><div class="reference">

	<b>Yan W, Hall AB, Jiang X.</b>
	2022.


	
	Bacteroidales Species in the Human Gut Are a Reservoir of Antibiotic Resistance Genes Regulated by Invertible Promoters.
	<i>Npj Biofilms and Microbiomes</i>.
	

	
	<b>8:</b>
	

	
	1.
	

	
	DOI: <a href="http://doi.org/10.1038/s41522-021-00260-1">10.1038/s41522-021-00260-1</a>.
	


	<div class="supp">
		
		<a href="/assets/pdf/2022_Yan.pdf">
			<i class="fa fa-file-pdf-o fa-lg"></i>
		</a>
		
		
		

		
			<a href="https://ftp.ncbi.nlm.nih.gov/pub/mgx/IP-ARG_assemblies/">
				<i class="fa fa-table fa-lg"></i>
			</a>
		

		

	</div>
</div>
</li>
			
		
			
				<li><div class="reference">

	<b>Olejniczak M, Jiang X, Basczok MM, Storz G.</b>
	2022.


	
	KH Domain Proteins: Another Family of Bacterial RNA Matchmakers.
	<i>Molecular Microbiology</i>.
	

	
	<b>117:</b>
	

	
	10-19.
	

	
	DOI: <a href="http://doi.org/10.1111/mmi.14842">10.1111/mmi.14842</a>.
	


	<div class="supp">
		
		<a href="/assets/pdf/2022_Olejniczak.pdf">
			<i class="fa fa-file-pdf-o fa-lg"></i>
		</a>
		
		
		

		

		

	</div>
</div>
</li>
			
		
			
				<li><div class="reference">

	<b>Mou Z, Yang Y, Hall AB, Jiang X.</b>
	2021.


	
	The Taxonomic Distribution of Histamine-Secreting Bacteria in the Human Gut Microbiome.
	<i>BMC Genomics</i>.
	

	
	<b>22:</b>
	

	
	695.
	

	
	DOI: <a href="http://doi.org/10.1186/s12864-021-08004-3">10.1186/s12864-021-08004-3</a>.
	


	<div class="supp">
		
		<a href="/assets/pdf/2021_Mou.pdf">
			<i class="fa fa-file-pdf-o fa-lg"></i>
		</a>
		
		
		

		

		

	</div>
</div>
</li>
			
		
			
				<li><div class="reference">

	<b>Braccia DJ, Jiang X, Pop M, Hall AB.</b>
	2021.


	
	The Capacity to Produce Hydrogen Sulfide (H2S) via Cysteine Degradation Is Ubiquitous in the Human Gut Microbiome.
	<i>Frontiers in Microbiology</i>.
	

	
	<b>12:</b>
	

	
	705583.
	

	
	DOI: <a href="http://doi.org/10.3389/fmicb.2021.705583">10.3389/fmicb.2021.705583</a>.
	


	<div class="supp">
		
		<a href="/assets/pdf/2021_Braccia.pdf">
			<i class="fa fa-file-pdf-o fa-lg"></i>
		</a>
		
		
		

		

		

	</div>
</div>
</li>
			
		
	</ol>
</div>

<p><a href="/research">backâ€¦</a></p>

</div>

	</div>

    <br/>

  </body>

</html>
