<!DOCTYPE html>
<html>

  <!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Code | The Jiang Lab at NLM</title>
<meta name="generator" content="Jekyll v3.9.2" />
<meta property="og:title" content="Code" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="We are a computational genomics group and develop methods and tools to analyze microbial and viral genomic data. All of our code is open source and available on Github. Click on the badges below to learn more about the methods we developed." />
<meta property="og:description" content="We are a computational genomics group and develop methods and tools to analyze microbial and viral genomic data. All of our code is open source and available on Github. Click on the badges below to learn more about the methods we developed." />
<link rel="canonical" href="https:///code/" />
<meta property="og:url" content="https:///code/" />
<meta property="og:site_name" content="The Jiang Lab at NLM" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2023-08-10T11:48:33-05:00" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Code" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2023-08-10T11:48:33-05:00","datePublished":"2023-08-10T11:48:33-05:00","description":"We are a computational genomics group and develop methods and tools to analyze microbial and viral genomic data. All of our code is open source and available on Github. Click on the badges below to learn more about the methods we developed.","headline":"Code","mainEntityOfPage":{"@type":"WebPage","@id":"https:///code/"},"url":"https:///code/"}</script>
<!-- End Jekyll SEO tag -->

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width initial-scale=1" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>Code</title>
    <meta name="description" content="The website for the bioinformatic research group leading by Xiaofang Jiang at the National Library of Medicine, National Institutes of Health.">

	<link rel="stylesheet" type='text/css' href="/css/bootstrap.min.css">
	<link rel="stylesheet" type='text/css' href="/css/font-awesome.min.css">

	<link rel="stylesheet" href="/css/academicons.css" />

	<link rel="stylesheet" type='text/css' href="/css/main.css">

	<link href='https://fonts.googleapis.com/css?family=Ubuntu' rel='stylesheet' type='text/css'>

	<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" rel="stylesheet"  type='text/css'>

    <link rel="canonical" href="https:///code/">
    <link rel="alternate" type="application/rss+xml" title="The Jiang Lab at NLM" href="https:///feed.xml" />

   
</head>


  <body>

	

<div class="header">
	<div class="container-fluid">
		<div class="pull-right">
			<ul class="nav nav-pills">
				
				<li role="presentation" ><a href="/">Home</a></li>
				
				<li role="presentation" ><a href="/research">Research</a></li>
				<li role="presentation" ><a href="/people">Team</a></li>
				<li role="presentation" ><a href="/papers">Papers</a></li>
				<li role="presentation" class="active"><a href="/code">Code</a></li>
				<li role="presentation" ><a href="/contact">Contact</a></li>
			</ul>
		</div>
	</div>
</div>


	<div class="default-wrapper">
	    <div class="index_page">
	<h2>Code</h2>
    <p>We are a computational genomics group and develop methods and tools to analyze microbial and viral genomic data. All of our code is open source and available on
<a href="https://github.com/nlm-irp-jianglab">Github</a>. Click on the badges below to learn more about the methods we developed.</p>

<hr />

<h2 id="evolink-">Evolink <a href="https://github.com/nlm-irp-jianglab/Evolink"><img src="../assets/img/Evolink.jpg" class="pic" align="right" height="180" /></a></h2>

<p>A python script for rapid identification of genotype-phenotype associations in large-scale microbial multi-species data.</p>

<p><a href="https://github.com/nlm-irp-jianglab/Evolink"><img src="https://img.shields.io/static/v1?style=flat&amp;logo=GitHub&amp;label=+&amp;message=GitHub&amp;color=black" alt="GitHub" /></a>
<a href="https://doi.org/10.1093/bioinformatics/btad215"><img src="https://img.shields.io/static/v1?style=flat&amp;logo=google-scholar&amp;label=+&amp;message=Paper&amp;color=white" alt="Paper" /></a></p>

<p>Evolink is a phylogeny-based tool that detects genotypes associated with a phenotype in multi-species datasets by using phylogenetic information to control for population structure, avoiding spurious findings. It outperforms similar tools in accuracy and efficiency.</p>

<hr />

<h2 id="super-">SuPER <a href="https://github.com/nlm-irp-jianglab/SuPER"><img src="../assets/img/SuPER.png" class="pic" align="right" height="139" /></a></h2>

<p>A command-Line tool to identify TRS-B sites in coronavirus genomes.</p>

<p><a href="https://github.com/nlm-irp-jianglab/SuPER"><img src="https://img.shields.io/static/v1?style=flat&amp;logo=GitHub&amp;label=+&amp;message=GitHub&amp;color=black" alt="GitHub" /></a>
<a href="https://doi.org/10.1093/molbev/msaa281"><img src="https://img.shields.io/static/v1?style=flat&amp;logo=google-scholar&amp;label=+&amp;message=Paper&amp;color=white" alt="Paper" /></a></p>

<p>SuPER is used to identify TRS-B sites in coronavirus genomes. SuPER first uses a covariance model derived from Rfam to identify TRS-L via profile-based sequence and structure scoring. Then, SuPER identifies TRS-B sites either by identifying template switching junctions using RNA-seq data or in by identifying sequences preceding genes that are found to be similar to the TRS-L CS sites.</p>

<hr />

<h2 id="cov-dist-">CoV-Dist <a href="https://github.com/nlm-irp-jianglab/CoV-Dist"><img src="../assets/img/CoV-Dist.png" class="pic" align="right" height="90" /></a></h2>

<p>A python script to measure the dissimilarity distance between SARS-CoV-2 metagenome samples.</p>

<p><a href="https://github.com/nlm-irp-jianglab/CoV-Dist"><img src="https://img.shields.io/static/v1?style=flat&amp;logo=GitHub&amp;label=+&amp;message=GitHub&amp;color=black" alt="GitHub" /></a>
<a href="https://doi.org/10.1016/j.watres.2021.117710"><img src="https://img.shields.io/static/v1?style=flat&amp;logo=google-scholar&amp;label=+&amp;message=Paper&amp;color=white" alt="Paper" /></a></p>

<p>CoV-Dist was developed to fill the need for an approach to perform pairwise comparisons of amplicon sequenced SARS-CoV-2 samples. This tool calculates the Yue and Clayton dissimilarity index of SARS-CoV-2 metagenomic samples relative to
a reference sequence and outputs a distance matrix that can be used for sample level comparisons and visualization.</p>

<hr />

<h2 id="phasefinder-">PhaseFinder <a href="https://github.com/nlm-irp-jianglab/PhaseFinder"><img src="../assets/img/PhaseFinder.png" class="pic" align="right" height="139" /></a></h2>

<p>A command-Line Tool for detection of DNA inversion mediated phase variation in bacterial genomes.</p>

<p><a href="https://github.com/nlm-irp-jianglab/PhaseFinder"><img src="https://img.shields.io/static/v1?style=flat&amp;logo=GitHub&amp;label=+&amp;message=GitHub&amp;color=black" alt="GitHub" /></a>
<a href="https://doi.org/10.1126/science.aau5238"><img src="https://img.shields.io/static/v1?style=flat&amp;logo=google-scholar&amp;label=+&amp;message=Paper&amp;color=white" alt="Paper" /></a></p>

<p>PhaseFinder is designed to detect DNA inversion mediated phase variation in bacterial genomes using genomic or metagenomic sequencing data. It works by identifying regions flanked by inverted repeats, mimicking their inversion in silico, and identifying regions where sequencing reads support both orientations.</p>

<hr />

<h2 id="immedb-">ImmeDB <a href="http://jianglabnlm.com/immedb/"><img src="../assets/img/ImmeDB.png" class="pic" align="right" height="120" /></a></h2>

<p>A database dedicated to the collection, classification, and annotation of mobile genetic elements (MGEs) from gut microbiome.</p>

<p><a href="http://jianglabnlm.com/immedb/"><img src="https://img.shields.io/static/v1?style=flat&amp;label=Docs&amp;message=Website&amp;color=success" alt="Website" /></a>
<a href="https://doi.org/10.1371/journal.pone.0223680"><img src="https://img.shields.io/static/v1?style=flat&amp;logo=google-scholar&amp;label=+&amp;message=Paper&amp;color=white" alt="Paper" /></a></p>

<p>ImmeDB (Intestinal microbiome mobile element) is comprised of all MGEs identified with a “deletion-based” method. This novel method allows identifying the exact mobilizable unit of active MGEs. In total 4928 putative MGEs were identified and classified into seven classes: integrative conjugative elements, integrative mobilizable elements, prophages, group II introns, transposons, unclassified genomic islands and unclassified islets.</p>

</div>

	</div>

    <br/>

  </body>

</html>
