<h2>TailspikeDB</h2>
<p><strong>TailspikeDB</strong> (tailspike protein database) is a database dedicated to the collection, classification, and annotation of prophage tailspike proteins from five bacteria pathogen genomes. In addition, this database provides the associations between tailspike proteins and bacterial surface glycans (determining the bacterial serotypes).</p>
<div><img src = "image/network.png" style = "margin:20px 20px 20px 20px;width: 70%;max-width: 70%;"></img></div>
<!--<h2>Classes of Mobile Genetic Elements in Gut Microbiome</h2>
<p><strong>Prophages</strong><span>: the latent form of lysogenic bacteriophages that autonomously transfer DNA between cells via transduction. Lysogenic bacteriophages insert into genomes via integrase and encode genes for a capsid which allows them to exit the host cell, and invade other cells .</span></p>
<p><strong>Integrative conjugative elements</strong><span> (ICEs): a class of MGE that can integrate into genomes using integrase, excise from the genome, circularize using relaxase, replicate, and then autonomously transfer to other cells using a set of conjugation-genes similar to those found in a plasmid.</span></p>
<p><strong>Integrative mobilizable elements</strong><span> (IMEs): a class of MGE with similarities to ICEs; they encode an integrase and relaxase for circularization, but lack genes for autonomous conjugation, and rely on other elements such as co-resident ICEs or conjugative plasmids to transfer between cells.</span></p>
<p><strong>Group II introns</strong><span>: a class of self-splicing introns that encode a protein capable of both reverse transcription of the spliced intron, and integration of the resulting DNA into chromosomes .</span></p>
<p><strong>Transposons</strong><span>: elements mobilized by transposases, including insertion sequences, which are simple transposons containing only genes essential for transposition, and composite transposons, which are mobilized sequence units flanked by two insertion sequences.</span></p>
<p><strong>Genomic islands/islets</strong><span>: discrete DNA segments differing between closely related bacterial strains that do not fall into traditional classes of MGEs.</span></p>
<p><span>In this study, the term "unclassified genomic islands" and "unclassified genomic islets" were used to describe mobilizable units with an unidentified mechanism for integration and transfer that are greater than 10kb and less than 10kb respectively.</span></p>
<p><span>Download link for ImmeDB : -->
<!--</span><a href="data/Data2_Annotation_of_genes_on_MGEs.gff3" style="font-size: medium; ">Annotations</a><span> (gff);
</span><a href="data/Data1_MGE_sequences.fasta" style="font-size: medium; ">Genomic sequences</a><span> (fasta)</span></p>-->
